### PR TITLE
Wide points may or may not be clipped outside the x/y clip planes.

### PR DESCRIPTION
**Pull request by jdashg**

### CODE REVIEW

**Comment by jdashg** [comment]
Resolves #2888.

File: specs/latest/1.0/index.html [hunk 1]
```diff
@@ -4394,6 +4394,32 @@ <h3>Uniform and attribute name collisions</h3>
...
+    <code>POINTS</code> primitives may or may not be discarded if the vertex lies outside the clip
+    volume, but within the near and far clip planes.
+</p>
+<div class="note rationale">
+    <p>
+        Clipping of wide points works differently in GLES and GL, and this difference in behavior is
+        prohibitive to work around in implementations.
+    </p>
+    <p>
+        OpenGL ES 2.0.25 p46:
```
Comment (kenrussell) [inline specs/latest/1.0/index.html]: Possible to add a hyperlink to https://www.khronos.org/registry/OpenGL/specs/es/2.0/es_full_spec_2.0.pdf ?

File: specs/latest/1.0/index.html [hunk 2]
```diff
@@ -4394,6 +4394,32 @@ <h3>Uniform and attribute name collisions</h3>
...
+        <blockquote>
+            If the primitive under consideration is a point, then clipping discards it if it lies
+            outside the near or far clip plane; otherwise it is passed unchanged.
+        </blockquote>
+    </p>
+    <p>
+        OpenGL 3.2 Core p97:
```
Comment (kenrussell) [inline specs/latest/1.0/index.html]: Possible to link to https://www.khronos.org/registry/OpenGL/specs/gl/glspec32.core.pdf ? (Or perhaps reference the 4.1 spec, https://www.khronos.org/registry/OpenGL/specs/gl/glspec41.core.pdf , which is the latest one supported on macOS?)

**Comment by jdashg** [comment]
@kenrussell PTAL

**Pull request merged**